alf/drivers/policy_driver.py

```diff
@@ -221,6 +221,14 @@ def get_initial_state(self):
         return common.zero_tensor_from_nested_spec(self._policy_state_spec,
                                                    self.env.batch_size)
 
+    def get_step_metrics(self):
```

Comment:
It's unclear what metrics are returned.
Make the comment more specific, the two metrics are episodes and environmental steps.

Reply:
updated